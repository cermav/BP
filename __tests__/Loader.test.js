import React from "react";
import ReactDOM from "react-dom";
import Loader from "../components/Loader";
import { render, cleanup, asFragment } from "@testing-library/react";

afterEach(cleanup);

it("renders without crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(<Loader />, div);
});

it("matches previous snapshot", () => {
    const { asFragment } = render(<Loader />);
    expect(asFragment()).toMatchSnapshot();
});
