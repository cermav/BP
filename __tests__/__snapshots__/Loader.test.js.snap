// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches previous snapshot 1`] = `
<DocumentFragment>
  <div
    class="loader"
  />
</DocumentFragment>
`;
